boudicca.base/common-model/src/main/kotlin/base/boudicca/model/structured/Key.kt

```diff
@@ -63,4 +63,6 @@ data class Key(val name: String, val variants: List<Variant>) : Comparable<Key>
             return Key(propertyName, variants.toList().sorted())
         }
     }
+
+    override fun toString() = toKeyString()
```

Comment:
what do we need the `toKeyString()` for, would `toString()` not be enough?

Reply:
toString() normally is for debug-logging and not for serializing stuff into whatever technical format, which is why i feel uncomfortable using toString() instead of toKeyString() when i want to make sure it corresponds to the right format. here it is coincidence that toString() does the same as toKeyString(), which is why i delegate it